modules/dnn/src/layers/elementwise_layers.cpp

```diff
@@ -577,12 +578,29 @@ struct ReLU6Functor : public BaseFunctor
     }
 #endif  // HAVE_DNN_NGRAPH
 
+
+
 #ifdef HAVE_WEBNN
+    ml::Operand BuildConstant(const ml::GraphBuilder& builder,
```

Comment:
could you please refer to https://github.com/webmachinelearning/webnn-native/blob/main/examples/SampleUtils.h#L43?